docs/torchserve_on_win_native.md

```diff
@@ -48,9 +48,11 @@ NOTE At present, wheels for windows are not available on PyPi. However following
  - Start 'Anaconda Powershell Prompt' (APP) as Admin User i.e. By right click on APP and run following commands
  - `git clone https://github.com/pytorch/serve.git`
  - `pip install click`
- - `cd serve`
- - `python .\ts_scripts\install_dependencies.py`
- - `python .\ts_scripts\install_from_src.py`
+ - `cd serve`  
+ -  for development purposes, use command below:  
+   `python .\ts_scripts\install_dependencies.py --environment=dev`
```

Comment:
@maheshambule  Is there an analogous command for production environment for this step?

If not to reduce confusion please list the full sequence of steps of Dev and Prod as two separate subsections in the readme.

Reply:
Yes. It is there. The param --environment=prod is for production. The default value for environment is 'prod' only.
I have created two subsections to avoid confusion and provided explicit environment param for production.